repl.py

```diff
@@ -363,13 +366,18 @@ def repl(symbols):
         return None
 
 def print_usage(prog_name, out):
-    out.write("Usage: {} [options...] <auth.json>\n")
+    out.write("Usage: {} [options...] [auth.json]\n")
     out.write("\n")
-    out.write("    <auth-json>: See ReadMe.md for information on how to create this file.\n")
+    out.write("    [auth_json]: See ReadMe.md for information on how to create this file. (Default: <script_dir>/auth.json)\n")
```

Comment:
Is "auth_json" supposed to be "auth.json"?

Reply:
yes, not sure why I added the underscore here